packages/widgets/Source/I3SBuildingSceneLayerExplorer/I3SBuildingSceneLayerExplorer.css

```diff
@@ -1,30 +1,30 @@
-ul {
+.cesium-viewer-i3s-explorer ul {
   list-style-type: none;
 }
 
-.layersList {
+.cesium-viewer-i3s-explorer .layersList {
   padding: 0;
 }
```

Comment:
This could be made a bit cleaner using the new ability for nested CSS Selectors.

```css
.cesium-viewer-i3s-explorer {
  ul {
    list-style-type: none;
  }

  .layersList {
    padding: 0;
  }
  /* ... */
}
```

However once again brings up the question of what browsers/environments do we target support for? It was only generally available in the majority of browsers last December [as seen on caniuse](https://caniuse.com/?search=nesting)

Not a reason to hold up this PR but wanted to bring it up